lib/key/info.js

```diff
@@ -58,37 +59,37 @@ const primaryUser = async (key, date) => {
 
     const cert = primary.selfCertification;
     return {
-        userId: primary.user.userId.userid,
+        userId: primary.user.userID.userid,
         symmetric: cert.preferredSymmetricAlgorithms ? cert.preferredSymmetricAlgorithms : [],
         hash: cert.preferredHashAlgorithms ? cert.preferredHashAlgorithms : [],
         compression: cert.preferredCompressionAlgorithms ? cert.preferredCompressionAlgorithms : []
     };
 };
 
 export default async function keyInfo(rawKey, email, expectEncrypted = true, date = serverTime()) {
-    const keys = await getKeys(rawKey);
+    const key = await getKey(rawKey);
 
-    const algoInfo = keys[0].getAlgorithmInfo();
-    const mainFingerprint = keys[0].getFingerprint();
+    const algoInfo = key.getAlgorithmInfo();
+    const mainFingerprint = key.getFingerprint();
 
     const obj = {
-        version: keys[0].primaryKey.version,
-        publicKeyArmored: keys[0].toPublic().armor(),
+        version: key.keyPacket.version,
+        publicKeyArmored: key.toPublic().armor(),
         fingerprint: mainFingerprint, // FIXME: deprecated, use fingerprints instead
```

Comment:
remove `fingerprint`?

Reply:
Yeah, I think we can :+1: In general I think this function should eventually be replaced with individual functions that return specific info, but ok, we can do that later.